docs/blogs/Overview of Owasp-VulnerableApp.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Why VulnerableApp
```

Comment:
Title is still same, can you please change this. Also can you change the name of file to `Overview_of_Owasp-VulnerableApp.md` as space in file name is little problematic.

Reply:
changed title and renamed the file, pls check it @preetkaran20